client/src/assets/fonts.scss

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'Open Sans';
```

Comment:
your structure of the project should be next
assets
     fonts -- here should be fonts (otf, woff, woff2 etc). But this file should be on the same level where u have 
                   variables and general styles
     images